main.nf

```diff
@@ -163,215 +77,63 @@ process uncompressInputVCF {
-    <(grep -v '^>' variants_reads.fa) > temp.txt
-
-    # Reformat the fasta ID: inconsistencies in the separators, and no new line before the sequence mean that this next 
-    # command is a bit ugly
-    # Input:
-    # >[chr]|[pos interval]   [REF]       [ALT]       [rsID]    [QUAL] [FILT] [INFO]      [seq]
-    # Replace all spaces and tabs with "|":
-    # >[chr]|[pos interval]|[REF]|[ALT|[rsID]|[QUAL|[FILT]|[INFO]|[seq]
-    # Replace the last "|" with a space (this is between the header and the sequence):
-    # >[chr]|[pos interval]|[REF]|[ALT|[rsID]|[QUAL|[FILT]|[INFO] [seq]
-    # And finally replace the space with a newline:
-    # Output:
-    # >[chr]|[pos interval]|[REF]|[ALT|[rsID]|[QUAL|[FILT]|[INFO]
-    # [seq]
-    sed 's/\\t/|/g; s/ /|/g; s/\\(.*\\)|/\\1 /' temp.txt | tr ' ' '\\n' > variant_reads.out.fa
-    '''
-}
-
-/*
- * Align sequence with bowtie2
- */
-process alignWithBowtie {
-
-    // Memory required is 5 times the size of the fasta in Bytes or at least 1GB
-    memory Math.max(file(params.newgenome).size() * 5, 1073741824) + ' B'
-
-    input:
-        path "variant_reads.fa" from variant_reads_with_info
-        // This will get the directory containing the bowtie index linked in the directory
-        file 'bowtie_index' from newgenome_dir
-        // This ensures that bowtie index exists
-        val "unused" from bt2_done
-
-    output:
-        path "reads_aligned.bam" into reads_aligned_bam
-
-    """
-    bowtie2 -k 2 --np 0 -f -x bowtie_index/${newgenome_basename} variant_reads.fa | samtools view -bS - > reads_aligned.bam
-    """
-}
-
-/*
- * Sort the bam file with samtools and index the result.
- */
-process sortBam {
-
-    input:
-        path "reads_aligned.bam" from reads_aligned_bam
-
-    output:
-        path "reads_aligned.sorted.bam" into reads_sorted_bam
-        path "reads_aligned.sorted.bam.bai" into bam_index
-
-    """
-    samtools sort reads_aligned.bam -o reads_aligned.sorted.bam
-    samtools index reads_aligned.sorted.bam
-    """
-}
-
-/*
- * Take the reads and process them to get the remapped variants
- *
- */
-process readsToRemappedVariants {
+ * Sort VCF file
+ */
+process sortVCF {
 
     input:
-        path "reads_aligned.sorted.bam" from reads_sorted_bam
-        path "genome.fa" from params.newgenome
+        path "variants_remapped.vcf"
 
     output:
-        path "variants_remapped.vcf" into variants_remapped
+        path "variants_remapped_sorted.vcf", emit: variants_remapped_sorted
 
     """
-    # Ensure that we will use the reverse_strand.py from this repo
-    ${baseDir}/variant_remapping_tools/reads_to_remapped_variants.py -i reads_aligned.sorted.bam \
-        -o variants_remapped.vcf -f $params.flankingseq \
-        -s $params.scorecutoff -d $params.diffcutoff --newgenome genome.fa
+    cat variants_remapped.vcf | awk '\$1 ~ /^#/ {print \$0;next} {print \$0 | "sort -k1,1 -k2,2n"}' > variants_remapped_sorted.vcf
```

Comment:
https://github.com/EBIvariation/variant-remapping/pull/20#discussion_r600113314 applies here as well (I guess it's the same change)

Reply:
I moved the sorting of the VCF to a later stage so I could use bcftools.